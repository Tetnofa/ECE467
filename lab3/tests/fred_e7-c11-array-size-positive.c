int arr1[0];
int arr2[1];
int arr3[-1];
