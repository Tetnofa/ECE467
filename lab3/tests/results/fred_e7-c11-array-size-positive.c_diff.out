0a1,7
> Program [useIO=0](1,0)
> 	Array Declaration	Int*(1,0)
> 		Identifier[name:arr1](1,4)
> 	Array Declaration	Int[1](2,0)
> 		Identifier[name:arr2](2,4)
> 	Array Declaration	Int[-1](3,0)
> 		Identifier[name:arr3](3,4)
